### ISSUE TITLE
how to install Node in Mac

### ISSUE THREAD

**Issue by lohithkumarnadella** [nodeschool/discussions#1943]
I have node and npm versions but not able to "npm install -g javascripting"

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "javascripting"
npm ERR! node v7.2.1
npm ERR! npm  v3.10.10
npm ERR! path /usr/local/lib/node_modules
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access

npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
npm ERR!  { Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/lohithkumar/npm-debug.log

* node version: v7.2.1
* npm version:  v3.10.10

**Comment by zhdzmn** [comment]
you are trying a global install using the -g flag. and this will be installing in the /usr/local folder which is of a root permission. so you need to install global module as root

tldr;

try `sudo npm install -g javascripting`
